src/nav/nrql.yml

```
@@ -0,0 +1,73 @@
+title: NRQL
+path: /docs/nrql
+pages:
+  - title: Get started
+    pages:
+      - title: Get started with NRQL
+        path: /docs/nrql/get-started/introduction-nrql-new-relics-query-language
+      - title: How to query with NRQL
+        path: /docs/nrql/get-started/introduction-nrql-how-nrql-works    
+      - title: NRQL charts and dashboards
+        path: /docs/nrql/get-started/charts-and-dashboards-with-nrql
+      - title: NRQL syntax
+        path: /docs/nrql/nrql-syntax-clauses-functions       
+  - title: NRQL tutorials
+    pages:
+      - title: NRQL tutorial - Your first queries
+        path: /docs/nrql/nrql-tutorials/introduction-nrql-tutorial
+      - title: NRQL tutorial - Process your data
+        path: /docs/nrql/nrql-tutorials/nrql-tutorial-process-your-data
+      - title: NRQL tutorial - Advance your dashboarding 
+        path: /docs/nrql/nrql-tutorials/nrql-tutorial-advanced-dashboarding
+      - title: NRQL tutorial - Advanced functions
+        path: /docs/nrql/nrql-tutorials/nrql-tutorial-advanced-functions
+  - title: NRQL references
+    pages:
+      - title: Facet results by time
+        path: /docs/nrql/nrql-references/nrql-group-results-across-time
+      - title: NRQL math
+        path: /docs/nrql/nrql-references/nrql-math-using-select
+  - title: Query tools
+    pages:
+      - title: New Relic query builder
+        path: /docs/query-data/nrql-new-relic-query-language/query-tools/chart-builder-create-query-visualizations
+      - title: NerdGraph (GraphQL API)
+        path: /docs/query-data/nrql-new-relic-query-language/query-tools/nrql-api-queries
+  - title:  Explore NRQL
+    pages:
+      - title: Set a query's time range
+        path: /docs/nrql/using-nrql/query-time-range
+      - title: Query metric data
+        path: /docs/telemetry-data-platform/get-data/apis/query-metric-data-type
```

```suggestion
        path: /docs/telemetry-data-platform/get-data/apis/query-metric-data-type
```

re: the comments, this link